### DISCUSSION TITLE
Relationship between the Ops.Gl.Meshes length unit and canvas pixels

### DISCUSSION THREAD

**Discussion by ianww** [cables-gl/cables_docs#297] [Answers & Questions] [answered]
As best I can tell, at a scale of 1 the canvas width is always 6.4 length units: two cubes of width 3.2 side by side fill the canvas width exactly at all aspect ratios and at all manual resizings of the canvas.

Is the Ops.Gl.Meshes length unit set as a proportion of the canvas width as it appears to be, and is this a changeable setting?

**Comment by steam0r** [comment, marked as answer]
not really sure what you are aiming for. the length unit in the meshes is not really related to the canvas size at all, as it lives in 3d space. also the space it takes up on the canvas is really dependent on you camera position for example. 

maybe you want something like PixelProjection? https://cables.gl/op/Ops.Gl.PixelProjection

also, grids help with orientation and getting a feeling for scale:

https://cables.gl/op/Ops.Gl.Meshes.Grid
https://cables.gl/op/Ops.Gl.Meshes.FloorGrid

**Reply by ianww** [reply]
Great, thanks for that info!
